azurelinuxagent/common/protocol/goal_state.py

```diff
@@ -221,7 +221,7 @@ def __init__(self, xml_text):
                 tmp_file = prvs[pubkey]
                 prv = "{0}.prv".format(thumbprint)
                 os.rename(tmp_file, os.path.join(conf.get_lib_dir(), prv))
-                logger.info("Found private key matching thumbprint {0}".format(thumbprint))
+                logger.info("Found private key matching thumbprint with length: {0}".format(len(thumbprint)))
```

Comment:
The thumbprint itself shouldn't be a danger to log and is useful for debugging (needed it on an incident this week). Why are we changing this?

Reply:
@norakoiralamsft Yes, we shouldn't change this. As @jscalev points out, logging the thumbprint is not a concern, and it is needed for debugging some issues, especially issues related to Fast Track. The length is not enough to distinguish the certificate, since all the thumbprints are of the same length,